prerendering.bs

```diff
@@ -403,7 +403,9 @@ Every {{Document}} has an <dfn for="Document">activation start time</dfn>, which
 
      <p class="note">As with regular [=browsing contexts=], the [=prerendering browsing context=] can be [=discarded=] for any reason, for example if it becomes unresponsive, performs a restricted operation, or if the user agent believes prerendering takes too much resources.
 
-  1. Let |request| be a new [=request=] whose [=request/URL=] is |startingURL| and [=request/referrer policy=] is |referrerPolicy|.
+  1. Let |request| be a new [=request=] whose [=request/URL=] is |startingURL|, [=request/referrer policy=] is |referrerPolicy|, and [=request/policy container=] is |referrerDoc|'s [=Document/policy container=], [=clone a policy container|cloned=].
```

Comment:
|request|'s policy container is overwritten to a clone of [source browsing context](https://html.spec.whatwg.org/multipage/browsing-the-web.html#source-browsing-context)'s [active document](https://html.spec.whatwg.org/multipage/browsers.html#active-document)'s [policy container](https://html.spec.whatwg.org/multipage/dom.html#concept-document-policy-container) in Steps 10-11 of https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigate. So if |referrerDoc| is the active document of its browsing context, then this step is not needed.

Reply:
You're right! So only the activation part is needed.